lib/rack/tracker/handler.rb

```diff
@@ -68,7 +68,8 @@ def handler_name
   def tracker_options
     @_tracker_options ||= {}.tap do |tracker_options|
       options.slice(*allowed_tracker_options).each do |key, value|
-        if option_value = value.respond_to?(:call) ? value.call(env) : value
+        option_value = value.respond_to?(:call) ? value.call(env) : value
+        if !option_value.nil? 
```

Comment:
I personally don't like this style of `.nil?` check. 
But I believe understand why you did this... what do you think about something what Avdi recommends to do:

```ruby
if (option_value = value.respond_to?(:call) ? value.call(env) : value)
```

Reply:
The issue is that some values might be `false`, but we still want them to be added to the tracker_options. See https://github.com/railslove/rack-tracker/pull/123/files#diff-3357b6095421c732335692f9edb4ad2eR106

Reply:
Alternatives welcome

Reply:
Ah I see, now I understand :) 

maybe `unless option_value.nil?` then?

Reply:
or `unless (option_value = value.respond_to?(:call) ? value.call(env) : value).nil?`

but that's maybe a little bit harder to read